File: Code/RepairShop/wwwroot/app/helpers/job-form.lookups.ts
module App.Helpers
{
    export interface IJobFormLookups
    {
        clients: IKeyValueResult<string, string>[];
        conditions: IKeyValueResult<string, string>[];
        models: IKeyValueResult<string, string>[];
        repairReasons: IKeyValueResult<string, string>[];
        users: IKeyValueResult<string, string>[];
        workDone: IKeyValueResult<string, string>[];
    }
}

File: Code/RepairShop/Components/JobFormLookups.cs
namespace RepairShop.Components
{
    using System;
    using System.Collections.Generic;

    class JobFormLookups
    {
        public IEnumerable<KeyValueResult<string, string>> clients { get; set; }
        public IEnumerable<KeyValueResult<string, string>> conditions { get; set; }
        public IEnumerable<KeyValueResult<string, string>> models { get; set; }
        public IEnumerable<KeyValueResult<string, string>> repairReasons { get; set; }
        public IEnumerable<KeyValueResult<string, string>> users { get; set; }
        public IEnumerable<KeyValueResult<string, string>> workDone { get; set; }
    }
}
